go/service/teams.go

```diff
@@ -35,17 +35,36 @@ func NewTeamsHandler(xp rpc.Transporter, id libkb.ConnectionID, g *globals.Conte
 	}
 }
 
+func (h *TeamsHandler) assertLoggedIn(ctx context.Context) error {
```

Comment:
might be better to do 

    loggedIn := h.G().ExternalG().ActiveDevice.Valid()

which will ensure you have unlocked device keys (and avoids login_state).

libkb.IsLoggedIn() just checks the session.